docker/openqa/Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM registry.opensuse.org/devel/openqa/containers/openqa_dev:latest
 ENV LANG en_US.UTF-8
 
+RUN zypper in -y -C lsof
```

Comment:
that's actually a good idea as a workaround as long as https://github.com/os-autoinst/openQA/pull/1968#issuecomment-456857026 is not solved

@Martchus WDYT?

Reply:
Yes, but I suppose one would need to add the dependency to `docker/travis_test/Dockerfile`. And I belief only *after* that change has been merged itself the Docker image is rebuilt and used for further PRs.

So don't add the workaround here. And I'd say @coolo should just accept the submit request.

Reply:
I wanted to make sure that tests do pass and then either remove workaround from openqa/Dockerfile or maybe keep it just for some time and add lsof dependency into  travis_test/Dockerfile (?)

Reply:
It didn't work so I give up trying to inject the fix here